recipes/fluids/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "fluids" %}
+{% set version = "0.1.74" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bd91938b42cb03519b3a98e2e26af9e548cb9060ddd959420b2e9c07d53cc7c4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - numpy >=1.5.0
+    - pip
+    - python
+    - scipy >=0.9.0
+  run:
+    - numpy >=1.5.0
+    - python
+    - scipy >=0.9.0
+
+test:
+  imports:
+    - fluids
+
+about:
+  home: https://github.com/CalebBell/fluids
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Fluid dynamics component of Chemical Engineering Design Library (ChEDL)
+  description: |
+    Fluids is open-source software for engineers and technicians working in the fields of chemical, mechanical, or civil engineering. It includes modules for piping, fittings, pumps, tanks, 
```

Can you wrap text?

---

Sorry, @isuruf. This one I didn't understand (sorry, this is my first experience with conda recipes). There is one example I could based myself to perform this wrap?

---

The suggested is to insert new line characters to make them fit comfortably without sideways scrolling. Aiming for maximum line lengths of 80-100 characters is generally nice to have, i.e.:
```suggestion
    Fluids is open-source software for engineers and technicians working in
    the fields of chemical, mechanical, or civil engineering. It includes
    modules for piping, fittings, pumps, tanks, compressible flow, open-channel
    flow, and more.
```

---

Ah, nice! Thanks for the clarification, @chrisburr!